src/main.cpp

```diff
@@ -38,6 +38,10 @@
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 4, 5, U8X8_PIN_NONE);
 #elif HELTEC // display via i2c for Heltec board
 U8G2_SSD1306_128X64_NONAME_F_SW_I2C u8g2(U8G2_R0, 15, 4, 16);
+#elif TTGO18650 // display via i2c for TTGO18650
```

Comment:
these directives is the right form for enable/disable code. It's ok.

Reply:
Good